bonsai/ContentView.swift

```diff
@@ -8,10 +8,23 @@
 import SwiftUI
 
 struct ContentView: View {
```

Comment:
i think we can rename this View

Reply:
I think ContentView is supposed to be like a base app view, I've added some logic there just for testing purposes, so will remove before merging